src/diffpy/labpdfproc/data/wavelengths_config.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think this should just be in one of the python files.  We don't want to have to maintain a dtabase-like json file separately.  Is there a reason you moved this from tools.py?   This seems like unnecessary infrastructure for us to maintain.